include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -49,6 +49,10 @@ typedef struct oe_sgx_enclave_config_t
 
     /* XSave Feature Request Mask */
     uint64_t xfrm;
+
+    uint8_t config_id[64];
```

Comment:
Config_id/config_svn are not decided at enclave development time, nor should it be part of the enclave binary. Is `oe_sgx_enclave_config_t` set at enclave development time?